apis/database/v1alpha1/zz_generated.managed.go

```diff
@@ -0,0 +1,134 @@
+/*
```

Comment:
i really like how the managed resource interface methods can be generated now, that's pretty neat.  atta boy @negz! 🤓